src/main/java/org/zalando/nakadi/throttling/ThrottlingService.java

```diff
@@ -0,0 +1,118 @@
+package org.zalando.nakadi.throttling;
+
+import org.joda.time.Instant;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.zalando.nakadi.util.ZkConfigurationService;
+
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+@Component
+public class ThrottlingService {
+
+    private static final String BYTES_LIMIT = "nakadi.throttling.bytesLimit";
+    private static final String MESSAGES_LIMIT = "nakadi.throttling.messagesLimit";
+    private static final String BATCHES_LIMIT = "nakadi.throttling.batchesLimit";
+    private final long windowSize;
```

Comment:
This is time window? If it is time, then in English it will be `windowLength` or `windowDuration`.